docker-compose.yml

```diff
@@ -40,9 +40,9 @@ services:
       KEYCLOAK_API_CLIENT_SECRET_KEY: ${KEYCLOAK_API_CLIENT_SECRET_KEY}
 
       CSM_EMAIL_TO_SEND_FROM: ${CSM_EMAIL_TO_SEND_FROM}
+      CSM_EMAILS_TO_SEND_TO: ${CSM_EMAILS_TO_SEND_TO}

```

Comment:
Is passing this environment variable still necessary?

Reply:
It is actually. I added it back because we are now requiring the API related error emails to be configured through environment variables. We don't want org emails getting spammed with API errors and realistically no matter if you are an admin or not, only the developers will find API errors useful.